cmd/bee/cmd/db.go

```diff
@@ -199,121 +251,195 @@ func dbImportCmd(cmd *cobra.Command) {
+//			if err != nil {
+//				logger.Error(err, "getting sleep value failed")
+//			}
+//
+//			defer func() { time.Sleep(d) }()
+//
+//			dataDir, err := cmd.Flags().GetString(optionNameDataDir)
+//			if err != nil {
+//				return fmt.Errorf("get data-dir: %w", err)
+//			}
+//			if dataDir == "" {
+//				return errors.New("no data-dir provided")
+//			}
+//
+//			logger.Warning("starting to nuke the DB with data-dir", "path", dataDir)
+//			logger.Warning("this process will erase all persisted chunks in your local storage")
+//			logger.Warning("it will NOT discriminate any pinned content, in case you were wondering")
+//			logger.Warning("you have another 10 seconds to change your mind and kill this process with CTRL-C...")
+//			time.Sleep(10 * time.Second)
+//			logger.Warning("proceeding with database nuke...")
+//
+//			localstorePath := filepath.Join(dataDir, "localstore")
+//			err = removeContent(localstorePath)
+//			if err != nil {
+//				return fmt.Errorf("localstore delete: %w", err)
+//			}
+//
+//			statestorePath := filepath.Join(dataDir, "statestore")
+//
+//			forgetOverlay, err := cmd.Flags().GetBool(optionNameForgetOverlay)
+//			if err != nil {
+//				return fmt.Errorf("get forget overlay: %w", err)
+//			}
+//
+//			forgetStamps, err := cmd.Flags().GetBool(optionNameForgetStamps)
+//			if err != nil {
+//				return fmt.Errorf("get forget stamps: %w", err)
+//			}
+//
+//			if forgetOverlay {
+//				err = removeContent(statestorePath)
+//				if err != nil {
+//					return fmt.Errorf("statestore delete: %w", err)
+//				}
+//				// all done, return early
+//				return nil
+//			}
+//
+//			stateStore, err := leveldb.NewStateStore(statestorePath, logger)
+//			if err != nil {
+//				return fmt.Errorf("new statestore: %w", err)
+//			}
+//
+//			logger.Warning("proceeding with statestore nuke...")
+//
+//			if err = stateStore.Nuke(forgetStamps); err != nil {
+//				return fmt.Errorf("statestore nuke: %w", err)
+//			}
+//			return nil
+//		}}
+//	c.Flags().String(optionNameDataDir, "", "data directory")
+//	c.Flags().String(optionNameVerbosity, "trace", "verbosity level")
+//	c.Flags().Bool(optionNameForgetOverlay, false, "forget the overlay and deploy a new chequebook on next bootup")
+//	c.Flags().Bool(optionNameForgetStamps, false, "forget the existing stamps belonging to the node. even when forgotten, they will show up again after a chain resync")
+//	c.Flags().Duration(optionNameSleepAfter, time.Duration(0), "time to sleep after the operation finished")
+//	cmd.AddCommand(c)
+//}
+
+//func removeContent(path string) error {
+//	dir, err := os.Open(path)
+//	if err != nil {
+//		return err
+//	}
+//	defer dir.Close()
+//
+//	subpaths, err := dir.Readdirnames(0)
+//	if err != nil {
+//		return err
+//	}
+//
+//	for _, sub := range subpaths {
+//		err = os.RemoveAll(filepath.Join(path, sub))
+//		if err != nil {
+//			return err
+//		}
+//	}
+//	return nil
+//}
+
+func MarshalChunkToBinary(c swarm.Chunk) ([]byte, error) {
```

Comment:
The marshal and unmarshal functions can be attached directly to the `swarm.Chunk` implementation.

Reply:
yeah, but it'll cause cyclic dependencies with `postage`. So I left it `db` package.